services/nomad/build/master.cfg

```diff
@@ -0,0 +1,324 @@
+# vim: set filetype=python:
+
+import configparser
+import json
+import shlex
+
+from twisted.internet import defer
+
+from buildbot.plugins import util, secrets, reporters, worker, schedulers
+from buildbot.plugins import steps
+
+ini = configparser.ConfigParser()
+ini.read('/local/config.ini')
+
+with open(ini['buildbot'].get('workers', '/local/workers.json')) as f:
+    js = json.load(f)
+    workers = js.get("workers", [])
+    builders = js.get("builders", [])
+
+# TODO
+userInfoProvider = util.ldapUserInfo(
+    uri='ldap://localhost:389',
+    accountBase='ou=entities,dc=netauth,dc=voidlinux,dc=org',
+    accountPattern='(uid=%(username)s)',
+    accountFullName='displayname',
+    groupBase='ou=groups,dc=netauth,dc=voidlinux,dc=org',
+    groupName='',
+)
+auth = util.RemoteUserAuth(userInfoProvider=userInfoProvider)
+authz = util.Authz(
+    allowRules=[
+        util.AnyEndpointMatcher(role="ops", defaultDeny=False),
+        util.AnyControlEndpointMatcher(role="ops"),
+    ],
+    roleMatchers=[
+        util.RolesFromGroups(groupPrefix="build-"),
+    ]
+)
+
+c = BuildmasterConfig = {
+    'buildbotNetUsageData': None,
+    'protocols': {'pb': {'port': ini['buildbot'].getint('worker-port', 9989)}},
+    'secretsProviders': [secrets.SecretInAFile(dirname="/local/secrets")],
+    'workers': [],
+    'change_source': [],
+    'collapseRequests': True,
+    'schedulers': [],
+    'builders': [],
+    'services': [],
+    'title': ini['buildbot'].get('title', 'Void Linux'),
+    'titleURL': ini['buildbot'].get('title-url', 'https://voidlinux.org/'),
+    'buildbotURL': ini['buildbot'].get('url', 'http://localhost:8010/'),
```

Comment:
Does this port need to be templated?

Reply:
no, but it needs to be replaced when pushing to prod

Reply:
disregard my last comment, this is the fallback value if not given in the `config.ini`, using python dictionary `.get(key, default)` syntax